contribs/github-bot/internal/requirements/reviewer.go

```diff
@@ -176,33 +223,33 @@ func (r *ReviewByTeamMembersRequirement) IsSatisfied(pr *github.PullRequest, det
 
 	// Check how many members of this team already reviewed this PR.
 	reviewCount := uint(0)
-	reviews, err := r.gh.ListPRReviews(pr.GetNumber())
-	if err != nil {
-		r.gh.Logger.Errorf("unable to check if a member of team %s already reviewed this PR: %v", r.team, err)
-		return utils.AddStatusNode(false, detail, details)
-	}
 
 	stateStr := ""
 	if r.desiredState != "" {
 		stateStr = fmt.Sprintf("%q ", r.desiredState)
 	}
 	for _, review := range reviews {
-		for _, member := range teamMembers {
-			if review.GetUser().GetLogin() == member.GetLogin() {
-				if desired := r.desiredState; desired == "" || desired == review.GetState() {
-					reviewCount += 1
-				}
-				r.gh.Logger.Debugf(
-					"Member %s from team %s already reviewed PR %d with state %s (%d/%d required %sreview(s))",
-					member.GetLogin(), r.team, pr.GetNumber(), review.GetState(), reviewCount, r.count, stateStr,
-				)
+		login := review.GetUser().GetLogin()
+		if containsUserWithLogin(teamMembers, login) {
+			if desired := r.desiredState; desired == "" || desired == review.GetState() {
+				reviewCount += 1
 			}
+			r.gh.Logger.Debugf(
+				"Member %s from team %s already reviewed PR %d with state %s (%d/%d required %sreview(s))",
```

Comment:
```suggestion
				"Member %s from team %s already reviewed PR %d with state %s (%d/%d required %s review(s))",
```

Reply:
I changed the format here more in general, so it's more readable in the code :)